frontend/cypress/integration/login/survey-list/survey-list.spec.js

```diff
@@ -0,0 +1,33 @@
+context('survey list', ()=> {
+  const login = {
+    email:'student@gmail.com',
+    password:'123456'
+  };
+
+  before(() => {
+    cy.visit(`${Cypress.config("baseUrl")}/login`)
```

Comment:
It would be better if we do not repeat `${Cypress.config("baseUrl")}` across our codebase. 
There is a way to use only  `cy.visit('/login')`